ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -957,6 +1032,11 @@ class NGraphEncapsulateOp : public OpKernel {
     int time_copy_output_tensors_to_host =
         copy_output_tensors_to_host.ElapsedInMS();
 
+    if (m_executable_can_create_tensor) {
+      m_executable_pipelined_tensors_map.at(ng_exec).return_tensors(
+          pipeline_idx);
+    }
```

Comment:
TODO: something RAII like would have been nice. so that we don't run the risk of forgetting to return borrowed items and then deadlocking